### DISCUSSION TITLE
Table Widget: Light mode only?

### DISCUSSION THREAD

**Discussion by mikerockett** [filamentphp/filament#6922] [General]
It seems the table widget doesn't support dark mode?

**Light Mode**

<img width="845" alt="image" src="https://github.com/filamentphp/filament/assets/4586280/952d6b04-9fa0-4c1f-87b2-8d5d62e54e77">

**Dark Mode**

<img width="827" alt="image" src="https://github.com/filamentphp/filament/assets/4586280/83907f70-67aa-4208-870d-e8a7d5754995">

**Comment by danharrin** [comment]
Hm no, have you published your tables config file or something?

**Reply by mikerockett** [reply]
I haven't published anything in relation to the tables package. Only the form components. That being said, after an upgrade, resource tables are now doing the same thing, which tells me something's up with my config somewhere. Let me dig a bit more and I'll get back to you on this.

**Reply by mikerockett** [reply]
Ok, so it seems I had to separately publish the tables config and set dark mode to true there as well. Seems like the component(s) need(s) to be adjusted to account for both configs?

**Reply by danharrin** [reply]
You shouldn't need to set dark mode explicitly. Did you remove the `MirrorConfig` middleware from the main Filament config file?

**Reply by mikerockett** [reply]
That did it – possible that I removed it accidentally. Thanks!
